### PR TITLE
Remove sqlite

### PR DESCRIPTION
**Pull request by wedamija**
We no longer support sqlite, so remove references from the codebase

### CODE REVIEW

File: src/sentry/testutils/helpers/query.py [hunk 1]
```diff
@@ -17,9 +15,6 @@ def parse_queries(captured_queries):
 
     for query in captured_queries:
         raw_sql = query['sql']
-        match = re.search(r"QUERY = (.+) - PARAMS", query['sql'])
```
Comment (wedamija) [inline src/sentry/testutils/helpers/query.py]: @HazAT Just wanted to confirm that this code is only for sqlite

**Comment by wedamija** [comment]
I noticed we use sqlite to generate the api docs, not really sure whether that should be removed too

**Comment by mattrobenolt** [comment]
@markstory do you know if we still generate API docs through this stuff?

We can/should also probably just make them use Postgres.

File: src/sentry/utils/dates.py [hunk 2]
```diff
@@ -69,7 +69,6 @@ def get_sql_date_trunc(col, db='default', grouper='hour'):
     conn = connections[db]
 
     engine = get_db_engine(db)
-    # TODO: does extract work for sqlite?
     if engine.startswith('oracle'):
```
Comment (mattrobenolt) [inline src/sentry/utils/dates.py]: lol oracle

**Comment by markstory** [comment]
> do you know if we still generate API docs through this stuff?

The API docs generation does use SQLite presently. I can update it to use postgres as supporting fewer databases is more important than the docs using SQLite.

**Comment by wedamija** [comment]
@markstory Mind mashing approve on this?

**Review by markstory — APPROVED** [review]

**Pull request merged**